run-me.py

```diff
@@ -0,0 +1,2 @@
+import webbrowser
+webbrowser.open('https://www.youtube.com/watch?v=dQw4w9WgXcQ')
```

Comment:
A joke so nice, we made it twice.

I hope nobody executes random python code on their machine without examining it first.